### PR TITLE
added download logs

### PR DESCRIPTION
**Pull request by shay23b**

### CODE REVIEW

File: ui_src/src/domain/sysLogs/components/logsWrapper/index.js [hunk 1]
```diff
@@ -127,7 +141,7 @@ const LogsWrapper = () => {
         <div className="logs-wrapper">
             <logs is="3xd">
                 <list-header is="3xd">
-                    <p className="header-title">Latest Logs</p>
+                    <p className="header-title">Latest Logs ({logs?.length})</p>
```
Comment (avrhamNeeman) [inline ui_src/src/domain/sysLogs/components/logsWrapper/index.js:144]: Remove ({logs?.length})

File: ui_src/src/domain/sysLogs/components/logsWrapper/index.js [hunk 2]
```diff
@@ -155,6 +169,12 @@ const LogsWrapper = () => {
                     )}
                     components={!stopLoad ? { Footer } : {}}
                 />
+
+                {logs?.length > 0 &&
+                    logs?.map((value, index) => {
+                        return;
+                    })}
+                {/* <button onClick={downloadLogs}>download logs</button> */}
```
Comment (avrhamNeeman) [inline ui_src/src/domain/sysLogs/components/logsWrapper/index.js:177]: ??

File: ui_src/src/domain/sysLogs/components/logsWrapper/index.js [hunk 3]
```diff
@@ -66,6 +66,20 @@ const LogsWrapper = () => {
                 setSeqNum(message_seq);
                 setLogs((users) => [...users, ...data.logs]);
             }
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    const downloadLogs = async () => {
```
Comment (avrhamNeeman) [inline ui_src/src/domain/sysLogs/components/logsWrapper/index.js:74]: Delete duplicate function

File: ui_src/src/domain/sysLogs/index.js [hunk 4]
```diff
@@ -33,11 +36,37 @@ const SysLogs = () => {
...
+            document.body.appendChild(link);
+            link.click();
+        } catch (error) {}
+    };
+
     return (
         <div className="logs-container">
             <header is="3xd">
-                <h1 className="main-header-h1">System Logs</h1>
+                <h1 className="logs-header-h1">System Logs </h1>
```
Comment (avrhamNeeman) [inline ui_src/src/domain/sysLogs/index.js:54]: We use the same class name over all main pages to prevent duplicate CSS code.

Comment (avrhamNeeman) [reply ui_src/src/domain/sysLogs/index.js:54]: So bring it back to "main-header-h1"

File: ui_src/src/domain/sysLogs/style.scss [hunk 5]
```diff
@@ -4,10 +4,21 @@
...
+    p {
         margin: 0;
     }
-    header{
+    header {
         margin-bottom: 25px;
+        display: flex;
+        justify-content: space-between;
+    }
+    logs-header-h1 {
```
Comment (avrhamNeeman) [inline ui_src/src/domain/sysLogs/style.scss:15]: Delete logs-header-h1

File: ui_src/src/domain/sysLogs/components/logsWrapper/index.js [hunk 6]
```diff
@@ -155,6 +157,11 @@ const LogsWrapper = () => {
                     )}
                     components={!stopLoad ? { Footer } : {}}
                 />
+
+                {logs?.length > 0 &&
```
Comment (avrhamNeeman) [inline ui_src/src/domain/sysLogs/components/logsWrapper/index.js:161]: ??

File: server/memphis_handlers_monitoring.go [hunk 7]
```diff
@@ -443,9 +483,15 @@ cleanup:
 		})
 	}
 
-	sort.Slice(resMsgs, func(i, j int) bool {
-		return resMsgs[j].TimeSent.Before(resMsgs[i].TimeSent)
-	})
+	if getAll {
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_monitoring.go:486]: it is looks like the same code in the if block and in the else block

File: server/memphis_handlers_monitoring.go [hunk 8]
```diff
@@ -322,6 +324,33 @@ func (mh MonitoringHandler) GetSystemLogs(c *gin.Context) {
 	c.IndentedJSON(200, response)
 }
 
+func (mh MonitoringHandler) DownloadSystemLogs(c *gin.Context) {
+	const amount = 100
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_monitoring.go:328]: this is not needed because we want all logs

File: server/memphis_handlers_monitoring.go [hunk 9]
```diff
@@ -322,6 +324,33 @@ func (mh MonitoringHandler) GetSystemLogs(c *gin.Context) {
...
+	const timeout = 3 * time.Second
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_monitoring.go:329]: increase this one to 20 seconds

File: server/memphis_handlers_monitoring.go (hunk 7 above)
Comment (shay23b) [reply server/memphis_handlers_monitoring.go:486]: opposite order of logs - see 'i' and 'j' switch places

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**